src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -60,6 +60,18 @@ export const cPlusPlusOptions = {
         [["with-struct", false], ["with-getter-setter", true]],
         "with-getter-setter"
     ),
+    wstring: new EnumOption(
+        "wstring",
+        "Store strings using Utf-16 std::wstring, rather than Utf-8 std::string",
+        [["use-string", false], ["use-wstring", true]],
+        "use-string"
+    ),
+    wstring: new EnumOption(
+        "conformance",
```

Comment:
What's the purpose of this option?  Is it required for the wide strings?

Reply:
Oh, I see it's in reference to #1024.  I believe this will generate incorrect code in many cases.  If you absolutely must have it, please make it `"secondary"`.

Reply:
I included a couple of bug fixes in with this change. This one fixes #1024, which I needed to be able to build (my application requires permissive mode). However, it looks like it's one or the other. I haven't found a fix that builds in both permissive and conformance mode. You could want to use wide strings in conformance mode, so I made them separate flags.